localenv/happy-life-bank/seed.yml

```diff
@@ -21,6 +21,12 @@ accounts:
     initialBalance: 2000
     path: accounts/planex
     postmanEnvVar: planexPaymentPointer
+  - name: 'Alice Smith'
+    path: accounts/asmith
```

Comment:
maybe `alias` is better than `path` but just went with what was there originally